lib/defaults/defaults.go

```diff
@@ -188,6 +188,10 @@ const (
 	// UploaderConcurrentUploads is a default number of concurrent
 	UploaderConcurrentUploads = 10
 
+	// ConcurrentStreams is the number of concurrent session upload streams
+	// that Auth Server supports.
+	ConcurrentStreams = 4
```

Comment:
Curious - how did you decide on 4 concurrent streams as being the default/limit?

Reply:
Just picked a random number.